editor/app/ems/[incident_number]/page.tsx

```diff
@@ -255,6 +307,21 @@ export default function EMSDetailsPage({
           />
         </Col>
       </Row>
+      <Row>
+        <Col sm={12} className="mb-3">
+          <RelatedRecordTable
+            records={nonCR3Crashes ? nonCR3Crashes : []}
+            isValidating={isValidatingNonCR3}
+            noRowsMessage="No crashes found"
+            header="Possible Non-CR3 matches"
+            columns={emsNonCR3Columns}
+            mutation=""
+            onSaveCallback={onSaveCallback}
```

Comment:
as of right now this table doesnt actually have any editable fields so this `onSaveCallback` isnt doing anything... but neither is the `onSaveCallback` in the DataCard above for the "Summary"? also `isValidating` isnt doing anything here or in the DataCard component either :thinking: 

could update `RelatedRecordTable` and `DataCard` to not require these props?

Reply:
thanks for flagging this. definitely some code smell here. i wonder if we've not fixed this because it will cause some typing headaches, but i'm all for making those props optional if you want to do that 🙏